suite/tests/CMakeLists.txt

```diff
@@ -2579,6 +2579,51 @@ use_DynamoRIO_extension(client.drbbdup-test.dll drmgr)
 use_DynamoRIO_extension(client.drbbdup-test.dll drreg)
 use_DynamoRIO_extension(client.drbbdup-test.dll drbbdup)
 
+if (X86 AND LINUX)
+  add_exe(client.drbbdup-meta-label-bb-test client-interface/drbbdup-meta-label-bb-test.asm)
+  set_target_properties(client.drbbdup-meta-label-bb-test PROPERTIES LINKER_LANGUAGE C)
+  append_link_flags(client.drbbdup-meta-label-bb-test "-nostartfiles -nodefaultlibs -static")
+
+  add_library(client.drbbdup-meta-label-bb-test.dll SHARED  client-interface/drbbdup-meta-label-bb-test.dll.c)
+  setup_test_client_dll_basics(client.drbbdup-meta-label-bb-test.dll)
+  use_DynamoRIO_extension(client.drbbdup-meta-label-bb-test.dll drmgr)
+  use_DynamoRIO_extension(client.drbbdup-meta-label-bb-test.dll drreg)
+  use_DynamoRIO_extension(client.drbbdup-meta-label-bb-test.dll drbbdup)
+
+  torunonly_ci(client.drbbdup-meta-label-bb-test client.drbbdup-meta-label-bb-test
+    client.drbbdup-meta-label-bb-test.dll client-interface/drbbdup-meta-label-bb-test.asm "" "-max_bb_instrs 0" "")
+endif (X86 AND LINUX)
```

Comment:
Make a macro or function to share these three code blocks?  They look identical except the test name.

Reply:
Done